lib/form/html.js

```diff
@@ -169,6 +169,11 @@ const renderer = (type, errors) => {
         required = `required--${input.required}`;
       }
 
+      // Add required text to label
+      if (label) {
```

Comment:
I'd also recommend you trying to run the tests locally to ensure they pass. This can be done by running `npm test` after installing the dependencies. You would have seen immediately that this fails.

You also need to include tests when adding new code, so please do so.

I also highly recommend you look at how input plugins are implemented (which this registers). Take, for instance, the HTML [number](https://github.com/punchcard-cms/input-plugin-number/blob/master/index.js#L32) and [quote](https://github.com/punchcard-cms/input-plugin-quote/blob/master/index.js#L59) input plugins and you'll see immediately that they both control their own `label` elements. Inputs can only have a single label, this information should be included as a text element with an [`aria-describeby`](http://webaim.org/techniques/forms/advanced#describedby) attribute on the input and ID on the description (will require some REGEX to actually dive in to the inputs to do this).

This needs to work for single input plugins as well as individual inputs in a multi-input input field (like quote)